### DISCUSSION TITLE
Testing drag and drop

### DISCUSSION THREAD

**Discussion by gee4vee** [microsoft/vscode-discussions#2210] [Extension Development QnA] [answered]
Is it possible to trigger a drag and drop in extension tests?

**Comment by robconery** [comment]
Can you add a bit more details as to what you're seeing now, what you want to see, and maybe an example?

**Comment by gee4vee** [comment]
I’m not seeing anything in particular. I don’t have an issue or anything. I’m trying to determine what are the possibilities in testing drag and drop when testing an extension.

**Comment by burkeholland** [comment]
@connor4312 may know on this one

**Comment by connor4312** [comment]
What are you expecting dragging and dropping tests should do?

**Reply by gee4vee** [reply]
Invoke a drag programmatically so I can test my drop code.

**Reply by connor4312** [reply, marked as answer]
There's no drag and drop action in testing right now.
